.github/workflows/slow-tests.yml

```diff
@@ -16,7 +16,12 @@ jobs:
 
     strategy:
       matrix:
-        testcontainer-version: [ 1.18.3, 1.19.3 ]
+        testcontainer-version: [ 1.19.4 ]
+        mysql-version: [ "mysql:latest" ]
+        # 1.18.3 does not work with mysql:latest (8.3.x).
+        include:
+          - testcontainer-version: 1.18.3
```

Comment:
Why do we keep testing against old, buggy versions of the library?
It is nice to be able to leave bugs in the past at some point.

Reply:
tc 1.18 is still the "previous minor". These tests exist mainly to ensure that when we adopt a new version of a library, we do not accidentally break users that are stuck on that version because e.g. their test framework requires it. Gives a somewhat smoother path to adoption. 

We leave that version behind as soon as tc 1.20 is released.